mipdb/bootstrap.sh

```diff
@@ -10,5 +10,6 @@ MONETDB_ADMIN_USERNAME = \"$MONETDB_ADMIN_USERNAME\"
 MONETDB_LOCAL_USERNAME = \"$MONETDB_LOCAL_USERNAME\"
 MONETDB_LOCAL_PASSWORD = \"$MONETDB_LOCAL_PASSWORD\"
 MONETDB_PUBLIC_USERNAME = \"$MONETDB_PUBLIC_USERNAME\"
-MONETDB_PUBLIC_PASSWORD = \"$MONETDB_PUBLIC_PASSWORD\"" > "/home/config.toml"
-tail -f /dev/null
\ No newline at end of file
+MONETDB_PUBLIC_PASSWORD = \"$MONETDB_PUBLIC_PASSWORD\"
+SQLITE_DB_PATH = \"$DATA_PATH/$WORKER_IDENTIFIER.db\"" > "/home/config.toml"
```

Comment:
I assume that naming the database after the nodename is to be able to develop locally?

I think it would be better to have a specific location, in order not to "search" for it if we need to.

We could have the `SQLITE_DB_PATH` as a config variable? WDYT?